samples/snippets/src/main/java/com/example/mediatranslation/TranslateFromMic.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.mediatranslation;
+
+// [START media_translation_translate_from_mic]
+
+import com.google.api.gax.rpc.ClientStream;
+import com.google.api.gax.rpc.ResponseObserver;
+import com.google.api.gax.rpc.StreamController;
+import com.google.cloud.mediatranslation.v1beta1.SpeechTranslationServiceClient;
+import com.google.cloud.mediatranslation.v1beta1.StreamingTranslateSpeechConfig;
+import com.google.cloud.mediatranslation.v1beta1.StreamingTranslateSpeechRequest;
+import com.google.cloud.mediatranslation.v1beta1.StreamingTranslateSpeechResponse;
+import com.google.cloud.mediatranslation.v1beta1.StreamingTranslateSpeechResult;
+import com.google.cloud.mediatranslation.v1beta1.TranslateSpeechConfig;
+import com.google.protobuf.ByteString;
+import java.io.IOException;
+import javax.sound.sampled.AudioFormat;
+import javax.sound.sampled.AudioInputStream;
+import javax.sound.sampled.AudioSystem;
+import javax.sound.sampled.DataLine;
+import javax.sound.sampled.TargetDataLine;
+
+public class TranslateFromMic {
+
+  public static void translateFromMic() throws IOException {
```

Comment:
Might be worth making this one just main since its from a microphone.

Reply:
okay